app/views/layouts/application.html.erb

```diff
@@ -7,11 +7,15 @@
 
     <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
     <%= javascript_pack_tag "application", "data-turbolinks-track": "reload" %>
+	
   </head>
 
   <body>
   <%= render "layouts/header" %> 
   <div class="container">
+    <% flash.each do |message_type, message| %>
+	  <div class="alert alert-<%= message_type %>"><%= message %></div> 
+	<% end %>
```

Comment:
```suggestion
    <% flash.each do |message_type, message| %>
	<div class="alert alert-<%= message_type %>"><%= message %></div> 
    <% end %>
```

Reply:
<img width="476" alt="comment" src="https://user-images.githubusercontent.com/68420253/90350192-e3bc1500-e06e-11ea-8d54-2a9fc620a4b8.png">
this is what I have on my laptop, I didn't change anything.I couldn't see what is wrong?

Reply:
You can use text editor to check it

Reply:
done